plugins/module_utils/network/sonic/config/port_breakout/port_breakout.py

```diff
@@ -175,6 +176,56 @@ def _state_deleted(self, want, have, diff):
 
         return commands, requests
 
+    def _state_replaced(self, want, have, diff):
+        """ The command generator when state is replaced
+
+        :param want: the additive configuration as a dictionary
+        :param obj_in_have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+
+        commands = diff
+        requests = self.get_modify_port_breakout_requests(commands, want)
+        if commands and len(requests) > 0:
+            commands = update_states(commands, "replaced")
+        else:
+            commands = []
+
+        return commands, requests
+
+    def _state_overridden(self, want, have, diff):
+        """ The command generator when state is overridden
+
+        :param want: the additive configuration as a dictionary
+        :param obj_in_have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        commands = []
+        requests = []
+        if not diff:
+            return commands, requests
+        commands = have
+        requests = self.get_delete_port_breakout_requests(commands, have)
+
+        if commands and len(requests) > 0:
+            send_requests(self._module, requests)
```

Comment:
This will not generate the required output logs to show what has been deleted. Instead, "update_states" should be invoked to show the deleted breakout modes. 

Also, Instead of directly invoking "send_requests" here, it would be more efficient to just  append the required deletion requests to the list of requests to be executed on exit from the "_state_overridden" function.